README.md

```diff
@@ -53,8 +53,16 @@ For more options for `jest` run `yarn test --help`.
 
 ## Configuration documentation
 
-Configuration file is located in  `public/config/env.js`
+Configuration file is located in  `public/config/env.js`. 
```

Comment:
It's ok to have env.js not as an example. In the next version we will get rid of it and there will be used configuration from `sonic`.
Details here: https://github.com/openware/baseapp/tree/master